src/win32/pthread/pthread_mutex_destroy.c

```diff
@@ -37,7 +37,9 @@
 #include "pthread.h"
 #include "implement.h"
 
-int getpid(void);
+#if REALM_UWP
+int _getpid(void);
+#endif
```

Comment:
the getpid() declaration was introduced in another UWP PR. This PR simply restores old behaviour on WIN32. For UWP this PR fixes the fact that getpid() is deprecated on UWP and replaced by _getpid